### DISCUSSION TITLE
My FFZ chat settings doesn't work

### DISCUSSION THREAD

**Discussion by icynstant** [tmarenko/twitch_chat_antiban#28] [Ideas]
For example: 7TV emotes aren't opened when I click on them, chat doesn't stop when I move my mouse

**Comment by tmarenko** [comment]
Unfortunately, I do not plan to support third-party chat settings. This plugin already has too many dependencies (7TV, FFZ, BTTV). It currently rebuilds the chat structure from scratch, so I don't really like the idea of imitating the functionality of 7TV or other chat extensions. Firstly, it's unlikely that I'll be able to access a user's settings at all, unless 7TV or another extension explicitly grants permission to my extension. And even if such permission were given, it would be difficult to support all the settings and their possible combinations.

Although both of your suggestions could potentially be added as distinct new features, I probably won't undertake this in the near future.
However, you or someone else is still welcome to contribute a pull request.

**Comment by lnoss** [comment]
I've just implemented a very basic way to support user chat settings with #21. I saw that BetterTV also saves part of its settings in the local storage, but I guess that replicating their features is not good idea as @tmarenko said.

But, for the current issue, I am sure we can add a simple `title` attribute on the `img` elements with basic info for each emote. Not too hard, and supported innately by browsers by hovering.

**Comment by tmarenko** [comment]
> it's unlikely that I'll be able to access a user's settings at all, unless 7TV or another extension explicitly grants permission to my extension

FYI it's been almost a year since I asked 7TV devs to remove my extension from their list as "incompatible", yet it's still there.
(asked today once again since they changed contact email)

```
> GET https://7tv.io/v3/config/extension

<  {
    "id": [
        "ipnllhnoiiclnoonckahfcpahgehgdgb"
    ],
    "issues": [
        {
            "platform": null,
            "severity": "CLASHING",
            "message": "This extension is wholly incompatible with 7TV's high-performance chat and breaks layout"
        }
    ]
  }
```
